src/netconf.c

```diff
@@ -795,12 +796,15 @@ np2srv_rpc_kill_cb(sr_session_ctx_t *session, uint32_t UNUSED(sub_id), const cha
 
 int
 np2srv_rpc_commit_cb(sr_session_ctx_t *session, uint32_t UNUSED(sub_id), const char *UNUSED(op_path),
-        const struct lyd_node *UNUSED(input), sr_event_t event, uint32_t UNUSED(request_id),
+        const struct lyd_node *input, sr_event_t event, uint32_t UNUSED(request_id),
```

Comment:
I think you can move this whole callback into `netconf_confirmed_commit.c`, it may save you some `ncc_*` functions that can be static afterwards.